subworkflows/local/kofamscan.nf

```diff
@@ -0,0 +1,28 @@
+//
+// Run KOFAMSCAN on protein fasta from orf_caller output
+//
+
+include { KOFAMSCAN     } from '../../modules/local/eukulele/kofamscan'
+include { DOWNLOAD_KODB } from '../../modules/local/eukulele/download_kofamscan_db'
```

Comment:
Why did you put them in the `eukulele` directory?
I think you should call the modules `KOFAMSCAN_SCAN` and `KOFAMSCAN_DOWNLOAD` respectively, and put them in the corresponding directories, e.g. `kofamscan/scan`. In the nf-core case this will be the directory where you have `main.nf` and `meta.yml`. With local modules that's a bit different of course, which asks the question whether it's better to move to nf-core ASAP.

Reply:
Yes, this is actually my mistake. I started to make a subworkflow but then I stopped. I can transfer the two modules in the subworkflow